public/assets/css/navbar_admin.css

```diff
@@ -0,0 +1,59 @@
+:root{
+    --full-relative: 100%;
+}
+
+.navAdmin .navbar {
+    position: relative;
+}
+
+.navAdmin::before {
+    position: absolute;
+    top: 0;
+    content: ' ';
+    width: var(--full-relative);
+    height: 6px;
+    background-color: var(--primary-color);
+}
+
+.navAdmin .navbar-nav {
+    width: var(--full-relative);
+    justify-content: space-around;
+}
+
+.navAdmin .btn-nav-admin {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0;
+    border-radius: 100px;
+    width: 90px;
+    height: 35px;
```

Comment:
Wouldn't it be better without magic numbers? Why not use css variables here like you did with "--full-relative"?

Reply:
A good practice not mastered. It's changed